spring-rabbit/src/main/java/org/springframework/amqp/rabbit/core/RabbitTemplate.java

```diff
@@ -666,6 +668,28 @@ public void setUserCorrelationId(boolean userCorrelationId) {
 		this.userCorrelationId = userCorrelationId;
 	}
 
+	/**
+	 * True if separate publisher connection(s) are used.
+	 * @return true or false.
+	 * @since 1.7.5
```

Comment:
`@since 1.7.6`, although looking at this complex solution I would stay away from back porting since we always have a workaround as really separately configured `CF` .

I understand that you still think about Rabbit Binder, but I have answer there for you how may we proceed further for the proper fix: https://github.com/spring-cloud/spring-cloud-stream-binder-rabbit/pull/115#issuecomment-351727547

Reply:
>at this complex solution

I don't know why you keep saying it's complex; it's rather simple, I think.

But OK; no backport; go ahead with the binder fix.